package.json

```diff
@@ -18,14 +18,19 @@
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
     "@types/react-redux": "^7.1.7",
+    "allotment": "^1.2.0",
```

Comment:
just a general comment allotment is a very unused package that doesn't have too much documentation or forks. We might need to add documentation of our own just in case or fork their project in the future